azext_edge/tests/conftest.py

```diff
@@ -6,14 +6,51 @@
 
 import os
 import sys
+from typing import NamedTuple
 
 import pytest
 import responses
 
 
+class MarkerDefinition(NamedTuple):
+    """Metadata for a pytest marker"""
+
+    name: str
+    description: str
+    edge_exclude: bool  # If True, prevents auto-marking test as 'edge'
+
+
+# Marker definitions with metadata
+MARKERS = [
+    MarkerDefinition("edge", "mark tests that run against edge / cluster (auto-applied to unmarked tests)", True),
+    MarkerDefinition("e2e", "mark end-to-end containerized tests (support bundles, checks)", False),
+    MarkerDefinition("rpsaas", "mark tests that are cloud-side", True),
+    MarkerDefinition("upgrade", "mark tests that will run az iot ops upgrade", True),
+    MarkerDefinition("init_scenario_test", "mark tests that will run az iot ops init", True),
+    MarkerDefinition("require_wlif_setup", "mark tests that require workload identity trust setup", True),
+    MarkerDefinition("long_running", "mark tests that take a long time to run", False),
+    MarkerDefinition("no_global_setup", "mark tests that will not use global setup", False),
```

Comment:
What was the use case for no_global_setup & what is using it now?

Reply:
I don't know what actually uses it (I don't think anything does) but it was introduced here:

https://github.com/Azure/azure-iot-ops-cli-extension/pull/609/files

If we don't plan to use it, we can remove it, I just figured it may have been important